pkg/config/templates/rke2-90-harvester-server.yaml

```diff
@@ -10,3 +10,4 @@ kubelet-arg:
 - {{ printf "%q" $arg }}
 {{- end }}
 {{- end }}
+audit-policy-file: /etc/rancher/rke2/config.yaml.d/92-harvester-kube-audit-policy.yaml
```

Comment:
If there are nodes being promoted to the server role, do they need to specify the file as well?
We create this config file when promoting an agent node to a server node. Ref: https://github.com/harvester/harvester/blob/47a3af235e41d37bc661af4f07d87b82c6b99345/deploy/charts/harvester/templates/configmap.yaml#L61-L69

If this only needs to be applied once, then we are good.

Reply:
Good catch.

Will the new PR in `Harvester` fullfill this task? please take a further look, thanks.

https://github.com/harvester/harvester/pull/2783